frontend/src/hooks/useTranslateQuality.ts

```diff
@@ -0,0 +1,24 @@
+import { Quality } from '@/bindings/definitions';
+import { useTranslation } from 'react-i18next';
+
+/**
+ * Returns a function that translates values of the Quality enum.
+ */
+export function useTranslateQuality() {
+  const { t } = useTranslation('enums', { keyPrefix: 'Quality' });
```

Comment:
Why implement it as a hook?

Reply:
useTranslation is also a hook.
This means it can either be called from react components or other hooks, otherwhise react would complain.